src/components/notification-icon.component.js

```diff
@@ -20,7 +20,7 @@ const mapStateToProps = state => ({
   notificationsCount: state.notifications.notificationsCount,
 });
 
-class NotificationIconComponent extends Component {
+export class NotificationIconComponent extends Component {
```

Comment:
Why does this need to be exported? Can you test `NotificationIcon` which is exported at the bottom of this file?

Reply:
Given NotificationIcon is this component but connected to a Redux store, the options were to either just test this inner component or to test NotificationIcon itself and mock out the store etc. I can look into the latter option.

Reply:
Although exporting and testing the unwrapped Component seems to be recommended by Redux: https://redux.js.org/docs/recipes/WritingTests.html#connected-components

Reply:
Yep, you're right! This should be exported.

Reply:
(not to self: we might want to make this a dumb component w/o access to redux state and do all that state management elsewhere)